tool/tsh/common/tsh.go

```diff
@@ -876,6 +876,8 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	app.Flag("relay", "Teleport relay address, \"none\" to explicitly disable the use of a relay, or \"default\" to use the cluster-provided address even if a different address was specified at login time.").Envar(relayEnvVar).StringVar(&cf.Relay)
 	app.Flag("nocache", "Do not cache cluster discovery locally.").Hidden().BoolVar(&cf.NoCache)
 	app.Flag("user", "Teleport user, defaults to current local user.").Envar(userEnvVar).StringVar(&cf.Username)
+	app.Flag("home-dir", "Directory where client state, credentials, and config are stored. Defaults to \"~/.tsh\".").Envar(types.HomeEnvVar).StringVar(&cf.HomePath)
+	app.Flag("config-path", "Path to file where global client configuration is stored. Defaults to \"/etc/tsh.yaml\".").Envar(globalTshConfigEnvVar).StringVar(&cf.HomePath)
```

Comment:
nit: the UX on `--config-path` is a little weird. by doc:
https://github.com/gravitational/teleport/blob/07fa1a72bbdd8e3859e11a00fe952f4b93b805fa/docs/pages/connect-your-client/tsh.mdx?plain=1#L967-L979

The user is allowed to override the global one but not the user-specific one .... well this is existing issue, just with flag now expose this more

WDYT to name the flag `--global-config-path`. And maybe mention behavior with the user-specific one in the description.

Reply:
Agreed it can be confusing. I tried to clarify without adding too much added verbosity:
```
	app.Flag("home-dir", "Directory where client state, credentials, and user-specific config are stored.")
	app.Flag("global-config-path", "Path to file where shared config is stored. Defaults to \"/etc/tsh.yaml\".")
```